docs/language/overview.md

```diff
@@ -1572,13 +1548,13 @@ the "in" operator, e.g.,
 
 ##### 7.2.1.6 Predicate Search Term
 
-Any Boolean-valued [function](functions/README.md) like `is()`, `has()`,
-`grep()` etc. and any [comparison expression](#comparisons)
+Any Boolean-valued [function](functions/README.md) like `is`, `has`,
+`grep` etc. and any [comparison expression](#62-comparisons)
```

Comment:
```suggestion
`grep`, etc. and any [comparison expression](#62-comparisons)
```

Reply:
@nwt: Good catch. I am also a fan of the Oxford comma!